main.rb

```diff
@@ -0,0 +1,59 @@
+require 'date'
+require './app'
+
+def display_menu
+  puts "\nWelcome to School Library App!"
+  puts "\nPlease choose an option by entering a number:"
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+def process_option(option, app)
+  menu = {
+    1 => :list_books,
+    2 => :list_people,
+    3 => :create_person,
+    4 => :create_book,
+    5 => :create_rental,
+    6 => :list_rentals_for_person,
+    7 => :exit
+  }.freeze
+
+  if menu.key?(option)
+    app.send(menu[option])
+    return true if menu[option] == :exit
+  else
+    puts 'Invalid option.'
+  end
+
+  false
+end
+
+# def main
+#   app = App.new
+
+#   loop do
+#     display_menu
+#     option = gets.chomp.to_i
+#     break if process_option(option, app)
+#   end
+# end
+
```

Comment:
- [ ] Impressive progress in developing the UI of your school library app. I'd like to provide a constructive suggestion to enhance your codebase further. It's crucial to maintain a clean and organized codebase, which leads me to recommend the removal of any commented-out `code` within your files.

    - In pursuit of an optimized and well-maintained code, I urge you to undertake the task of removing all commented-out `code`. This proactive step will not only improve the present state of your code but will also simplify future development and collaboration.

Happy coding! 😊🚀🍀